layout/mainmenu.xml

```diff
@@ -23,102 +23,92 @@
+			<Panel id="MainMenuInput" class="full">
 				
 				<!-- Fills the entire screen. This is used as the backdrop -->
-				<Panel id="MainMenuCore" blurrects="ChaosLoadingScreen">
-					<Panel id="MainMenuBackground">
+				<Panel id="MainMenuCore" class="full" blurrects="ChaosLoadingScreen">
+					<Panel id="MainMenuBackground" class="full">
 
 						<!-- Background video -->
-						<Panel id="MainMenuMovieParent">
+						<Panel id="MainMenuMovieParent" class="full">
 							<!-- Movie added/removed dynamically when changing the visibility of the main menu -->
 						</Panel>
 						<Panel id="MainMenuVanityParent">
 							<!-- Vanity is not used but the engine needs this to exist -->
 						</Panel>
 					</Panel>
 
-					<Panel id="MainMenuCampaignFrameParent">
-						<Frame id="MainMenuCampaignFrame"> 
-							<Panel class="Main">
-								<Panel class="MainMenuNavlistContainer">
+					<Panel id="MainMenuCampaignFrameParent" class="full">
+						<Frame id="MainMenuCampaignFrame" class="full"> 
+							<Panel class="full">
+								<Panel id="MainMenuPanel" class="full MainMenuNavlistContainer">
 									<Image id="MainMenuLogoDark"
 											class=""
 											src=""
 											defaultsrc="file://{images}/menu/p2ce/logo_title.png"
 											scaling="stretch-to-cover-preserve-aspect"/>
-
-									<TextButton id=""
-											class="MainMenuNavlistItem MainMenuOnly"
-											group="NavBar"
-											onactivate="$.DispatchEvent( 'P2CEMainMenuPlayMap', 'sp_p2ce_campaign_hub' );"
-											text="Start New Game" />
-
-									<TextButton id=""
-											class="MainMenuNavlistItem MainMenuOnly"
-											group="NavBar"
-											onactivate="$.DispatchEvent( 'P2CEMainMenuPlayMap', 'sp_a2_triple_laser' );"
-											text="Load Game" />
-
-									<TextButton id=""
-											class="MainMenuNavlistItem PauseMenuOnly"
-											group="NavBar"
-											onactivate="$.DispatchEvent( 'P2CEPauseMenuResume' );"
-											text="Resume Game" />
-
-									<TextButton id=""
-											class="MainMenuNavlistItem MainMenuOnly"
-											group="NavBar"
-											onactivate="MainMenuController.onWorkshopMenuButtonPressed();"
-											text="Manage Addons" />
-
-									<TextButton id=""
-											class="MainMenuNavlistItem"
-											group="NavBar"
-											onactivate="MainMenuController.onSettingsMenuButtonPressed();"
-											text="Settings" />
-
-									<TextButton id=""
-											class="MainMenuNavlistItem PauseMenuOnly"
-											group="NavBar"
-											onactivate="GameInterfaceAPI.ConsoleCommand( 'disconnect' );"
-											text="Exit to Main Menu" />
-									<TextButton id=""
-											class="MainMenuNavlistItem"
-											group="NavBar"
-											onactivate="MainMenuController.hideQuitMenu(false)"
-											text="Quit" />
-
+									<Panel id="NavListContainer" class="full flow-down" >
+										<TextButton id=""
+												class="ListItem MainMenuNavlistItem MainMenuOnly"
```

Comment:
You can use `@extend` from sass to automatically copy all of ListItem's rules to the MainMenuNavlistItem class